src/integTest/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPluginIntegrationTest.groovy

```diff
@@ -9,6 +9,10 @@ class DockerJavaApplicationPluginIntegrationTest extends AbstractIntegrationTest
 
     def setup() {
         project = ProjectBuilder.builder().withProjectDir(projectDir).build()
+
+        project.repositories {
```

Comment:
The test case doesn't require any external dependencies. No need to define a repository.